biz.aQute.bnd.reporter/test/biz/aQute/bnd/reporter/plugins/entries/bundle/ChecksumPluginTest.java

```diff
@@ -0,0 +1,35 @@
+package biz.aQute.bnd.reporter.plugins.entries.bundle;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Locale;
+
+import org.junit.Test;
+
+import aQute.bnd.osgi.Jar;
+import aQute.bnd.osgi.Processor;
+import biz.aQute.bnd.reporter.artifact.dto.ChecksumDTO;
+
+public class ChecksumPluginTest {
+
+	@SuppressWarnings("unchecked")
+	@Test
+	public void testChecksumPlugin() throws IOException {
+		final ChecksumPlugin plugin = new ChecksumPlugin();
+		final Jar jar = new Jar("jar", new File("testresources/componentsEntry/source.jar"));
```

Comment:
You should close the jar object. Put it in a try-with-resource.

Reply:
done, thank you.

Reply:
It still seems to be unclosed.

Reply:
done with many others